smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/middlewares/providers/HttpUrlPathProvider.kt

```diff
@@ -49,8 +48,14 @@ class HttpUrlPathProvider(
     }
 
     fun renderProvider(writer: SwiftWriter) {
-        writer.openBlock("extension \$N: \$N {", "}", inputSymbol, ClientRuntimeTypes.Middleware.Providers.URLPathProvider) {
-            writer.openBlock("public var urlPath: \$T {", "}", SwiftTypes.String) {
+        writer.openBlock("extension \$N {", "}", inputSymbol) {
+            writer.write("")
+            writer.openBlock(
+                "static func urlPathProvider(_ value: \$N) -> \$T {",
+                "}",
+                inputSymbol,
+                SwiftTypes.String,
+            ) {
```

Comment:
- Protocol removed
- Protocol property replaced with static method that conforms to `URLPathProvider<inputSymbol>`
- Kotlin cleanup